kiosk/models.py

```diff
@@ -15,8 +16,13 @@ class KioskItem(models.Model):
     active = models.BooleanField(default=True)
     media = models.FileField(upload_to='kiosk', null=False, validators=[validate_file_extension])
     ordering = models.IntegerField(null=False, default=random_ordering, blank=False)
+    start_datetime = models.DateTimeField(default=timezone.now(), blank=True)
+    end_datetime = models.DateTimeField(null=True, blank=True)
 
     @property
     def is_image(self):
         name, extension = os.path.splitext(self.media.name)
         return extension in valid_images
+
+    def is_active_period(self, current_time) -> bool:
+        return (self.start_datetime < current_time) and (self.end_datetime is None or self.end_datetime > current_time)
```

Comment:
```suggestion
    def is_active_period(self) -> bool:
        now = timezone.now()
        return (self.start_datetime <= now) and (self.end_datetime is None or self.end_datetime >= now)
```
No need to pass time to filter as arg. May be `@property` as well

Reply:
My thought process was that it could be used to introduce a mockup time in case of a test, but there's probably not much use for that.

Reply:
Look at our usage of `freezegun` for mocking time in tests: https://github.com/f-klubben/stregsystemet/blob/9748ebebecc7cdb8fca69457aad70fa1a86924a2/stregsystem/tests.py#L823

Not needed for this feature as the queries are so simple.